src/module/oscillator.rs

```diff
@@ -47,6 +57,7 @@ impl ModuleT for Oscillator {
             let t0 = (t + i as u64) as f64 / SAMPLE_RATE as f64;
             let sample: f32 = match &self.params.waveform {
                 Waveform::Sine => f64::sin(co * t0),
+                Waveform::Square => sign(f64::sin(co * t0)),
```

Comment:
I'd inline this if Rust supported open ranges in match statements, but that's not a stable feature.

This woulda been nice: 

``` rust
WaveForm::Square => match f64::sin(co * t0) {
  0.0 => 0.0,
  ..0.0 => -1.0,
  1.0.. => 1.0,
}